components/search_results/search_results.jsx

```diff
@@ -198,18 +199,19 @@ class SearchResults extends React.Component {
         } else if (this.props.isFlaggedPosts && noResults) {
             ctls = (
                 <div className='sidebar--right__subheader a11y__section'>
-                    <FlagPostSearchHint
-                        dataRetentionEnableMessageDeletion={this.props.dataRetentionEnableMessageDeletion}
-                        dataRetentionMessageRetentionDays={this.props.dataRetentionMessageRetentionDays}
+                    <NoResultsIndicator
+                        variant={NoResultsVariant.FlaggedPosts}
+                        formattedHTMLMessageValues={{icon: <FlagIcon className='icon  no-results__icon'/>
+                        }}
                     />
                 </div>
             );
         } else if (this.props.isPinnedPosts && noResults) {
             ctls = (
                 <div className='sidebar--right__subheader a11y__section'>
-                    <PinPostSearchHint
-                        dataRetentionEnableMessageDeletion={this.props.dataRetentionEnableMessageDeletion}
-                        dataRetentionMessageRetentionDays={this.props.dataRetentionMessageRetentionDays}
+                    <NoResultsIndicator
+                        variant={NoResultsVariant.PinnedPosts}
+                        formattedHTMLMessageValues={{boldText: <strong>{'Pin to Channel'}</strong>}}
```

Comment:
should the key here just be `text` as opposed to `boldText` since it's the caller that's making it bold rather than the `NoResultsIndicator`?

Reply:
can do, also need to change the prop name since its not an HTML message :D